controllers/ironic_controller.go

```diff
@@ -514,6 +514,7 @@ func (r *IronicReconciler) conductorDeploymentCreateOrUpdate(instance *ironicv1.
 		deployment.Spec.DatabaseHostname = instance.Status.DatabaseHostname
 		deployment.Spec.DatabaseUser = instance.Spec.DatabaseUser
 		deployment.Spec.Secret = instance.Spec.Secret
+		deployment.Spec.NodeSelector = instance.Spec.NodeSelector
```

Comment:
Ah damn, this shouldn't be changed here.. Should be updated in one of the other commits

Reply:
Fixing this in the 3rd commit is fine I think, as long as its in the overall PR.